README.md

```diff
@@ -1,9 +1,11 @@
-# The Pipage
+<h1 align="center">The Pipage</h1>
+<p align="center">
 [![npm](https://img.shields.io/npm/v/pipage.svg?style=flat-square)](https://npmjs.com/package/pipage)
 [![npm license](https://img.shields.io/npm/l/pipage.svg?style=flat-square)](https://npmjs.com/package/pipage)
 [![npm downloads](https://img.shields.io/npm/dm/pipage.svg?style=flat-square)](https://npmjs.com/package/pipage)
 [![build status](https://img.shields.io/travis/resin-io-modules/pipage/master.svg?style=flat-square&label=mac%20/%20linux)](https://travis-ci.org/resin-io-modules/pipage)
 [![build status](https://img.shields.io/appveyor/ci/resin-io/pipage/master.svg?style=flat-square&label=windows)](https://ci.appveyor.com/project/resin-io/pipage)
+</p>
```

Comment:
I dunno if it's just a 'preview problem', but when I click on the `View` button just above, the badges-markup gets displayed inline, instead of showing the badges?